python/mvdtool.cpp

```diff
@@ -237,4 +252,18 @@ PYBIND11_MODULE(mvdtool, mvd) {
     py::class_<SonataFile, std::shared_ptr<SonataFile>>(sonata, "File", file)
         .def(py::init<const std::string&>())
         ;
+    py::class_<TSVFile, std::shared_ptr<TSVFile>>(tsv, "File", file)
+        .def(py::init<const std::string&>())
+        ;
+    py::class_<TSVInfo>(tsv, "TSVInfo")
+        .def(py::init<>())
+        .def_readwrite("morphName", &TSVInfo::morphName)
```

Comment:
Do we need read-write? what about readonly?

Reply:
Yes, this should be read-only!